src/wp-content/themes/twentysixteen/functions.php

```diff
@@ -377,7 +377,8 @@ function twentysixteen_javascript_detection() {
  */
 function twentysixteen_scripts() {
 	// Add custom fonts, used in the main stylesheet.
-	wp_enqueue_style( 'twentysixteen-fonts', twentysixteen_fonts_url(), array(), null );
+	$font_version = ( 0 === strpos( (string) twentysixteen_fonts_url(), get_template_directory_uri() . '/' ) ) ? '20230328' : null;
```

Comment:
Can we use `str_starts_with` here?

Reply:
Unfortunately, no. These themes should work with older versions of both PHP and WordPress.